cli/internal/upgrade/terraform.go

```diff
@@ -139,6 +142,14 @@ func (u *TerraformUpgrader) ApplyTerraformMigrations(ctx context.Context, fileHa
 		return fmt.Errorf("terraform apply: %w", err)
 	}
 
+	// AttestationURL is only set for Azure. In contrast to the rest of the implementation,
+	// this is not generic and should be refactored when further upgrades / CSPs are needed.
```

Comment:
What do you have in mind that you want to refactor? Please describe the wanted change precisely, either in a ticket or the comment. Alternatively, remove the "refactor this" part of the comment.

While looking at the current flow this seem like the adequate place for CSP-specific code to me. It has to be located somewhere :P